### ISSUE TITLE
iou loss 问题

### ISSUE THREAD

**Issue by LeBron-Jian** [Megvii-BaseDetection/YOLOX#947]
@FateScript  你们的losses.py写的  loss = 1 - iou ** 2，正常不应该是loss = 1 - iou 吗？ 还是说在实际测试中前者效果更好

**Comment by FateScript** [comment]
Part of code are inspired from code [here](https://github.com/ruinmessi/ASFF/blob/0ff0e3393675583f7da65a7b443ea467e1eaed65/models/utils_loss.py#L60).

And yes, use `1 - iou ** 2` is better than  `1 - iou` in YOLOX

**Comment by Joker316701882** [comment]
@LeBron-Jian You may also refer to this [paper](https://arxiv.org/abs/2110.13675).

**Comment by LeBron-Jian** [comment]
thanks a million
